packages/site/src/components/Abstract.tsx

```diff
@@ -16,7 +16,7 @@ export function Abstract({
   if (!content) return null;
   return (
     <div className={className}>
-      <h2 id={id} className="mb-3 text-base font-semibold group">
+      <h2 id={id} className="mb-3 text-base font-semibold group not-prose">
```

Comment:
Making this not-prose will change the font which is not ideal. Can we just add `mt-0`?